communication/slack-guidelines.md

```diff
@@ -1,61 +1,35 @@
 ---
 title: "Slack Guidelines"
 description: |
-  The policies, procedures, and escalation mechanisms for the Slack platform,
+  These are the policies, procedures, and escalation mechanisms for the Slack platform,
   such as requesting channels, tokens, and how to report inappropriate content.
 ---
 
-<!-- omit in toc -->
-# Slack Guidelines
-
 Slack serves as the main communication platform for the Kubernetes community
-outside of the mailing lists. It’s important that conversations stays on topic
-in each channel, and that everyone abides by the [Code of Conduct][coc]. There
-are over 100,000 members who should all expect to have a positive experience.
-
-Chat is searchable and public. Do not make comments that you would not say on a
-video recording or in another public space. Please be courteous to others.
-
-- [Code of Conduct](#code-of-conduct)
-- [Admins](#admins)
-- [General Communication Guidelines](#general-communication-guidelines)
-  - [Workspace Channel History](#workspace-channel-history)
-  - [DM (Direct Message) Conversations](#dm-direct-message-conversations)
-  - [Specific Channel Rules](#specific-channel-rules)
-  - [Escalating and/or Reporting a Problem](#escalating-andor-reporting-a-problem)
-- [Should you have a channel on the Kubernetes Slack?](#should-you-have-a-channel-on-the-kubernetes-slack)
-- [Requesting a Channel](#requesting-a-channel)
-  - [Delegating Channel Ownership](#delegating-channel-ownership)
-- [Requesting a User Group](#requesting-a-user-group)
-- [Requesting a Bot, Token, or Webhook](#requesting-a-bot-token-or-webhook)
-- [Moderation](#moderation)
-  - [Admin Expectations and Guidelines](#admin-expectations-and-guidelines)
-  - [Sending Messages to the Channel](#sending-messages-to-the-channel)
-  - [Processing Slack Requests](#processing-slack-requests)
-    - [Processing Channel Requests](#processing-channel-requests)
-    - [Processing User Group Requests](#processing-user-group-requests)
-    - [Processing Bot, Token, or Webhook Requests](#processing-bot-token-or-webhook-requests)
-  - [Inactivating Accounts](#inactivating-accounts)
+outside of the mailing lists. It's important that conversations stay on topic in
+each channel, and that everyone abides by the [Code of Conduct][4]. There are
```

Comment:
```suggestion
each channel, and that everyone abides by the [Code of Conduct][coc]. There are
```
Not sure why the `[4]` was put in there, its broken on page view - this is for all the CoC links in this doc as wel

Reply:
Good catch. A lot of this happened when I ran it through the conversion from Google Docs to Markdown. It was messy! I didn't catch it all.